quickwit-doc-mapper/src/default_doc_mapper/default_mapper.rs

```diff
@@ -138,16 +139,35 @@ impl DefaultDocMapperBuilder {
 
         // Build the root mapping entry, it has an empty name so that we don't prefix all
         // field name with it.
```

Comment:
```suggestion
```
I feel this would be a more interesting info in the class doc than here where we're a click away from the implem